File: python/modelo/enfermero.py
from datetime import date
from modelo.personal_salud import PersonalSalud
class Enfermero(PersonalSalud):
    def __init__(self, dni: str, nombres: str, apellidos: str, fecha_nac: date, sexo: str,
                 colegiatura: str, programa: str = "CRED"):
        super().__init__(dni, nombres, apellidos, fecha_nac, sexo, colegiatura, "ENFERMERIA", "TARDE")
        self.__programa = programa
    def firmar_atencion(self, id_atencion: str) -> str:
        return f"FIRMA_CEP_{self._codigo_colegiatura}_TRIAGE_{id_atencion}"
    def obtener_rol(self) -> str:
        return "ENFERMERO"

File: java/modelo/Enfermero.java
package modelo;
import java.time.LocalDate;
public class Enfermero extends PersonalSalud {
    private String programaAsignado;
    public Enfermero(String dni, String nombres, String apellidos, LocalDate fechaNacimiento,
                     String sexo, String codigoColegiatura, String programaAsignado) {
        super(dni, nombres, apellidos, fechaNacimiento, sexo, codigoColegiatura, "ENFERMERIA", "TARDE");
        this.programaAsignado = programaAsignado;
    }
    @Override
    public String firmarAtencion(String idAtencion) {
        return "FIRMA_CEP_" + this.codigoColegiatura + "_TRIAGE_" + idAtencion;
    }
    @Override
    public String obtenerRol() {
        return "ENFERMERO";
    }
    public String getProgramaAsignado() {
        return programaAsignado;
    }
}